specification/under_development/profiles/Base_Profile.md

```diff
@@ -170,7 +171,7 @@ declare void @__quantum__rt__result_record_output(%Result*, i8*)
 
 ; attributes
 
-attributes #0 = { "entry_point" "qir_profiles"="base_profile" "output_labeling_schema"="schema_id" "required_num_qubits"="2" "required_num_results"="2" }
+attributes #0 = { "entry_point" "qir_profiles"="base_profile" "output_labeling_format"="format_id" "required_num_qubits"="2" "required_num_results"="2" }
```

Comment:
It came up during steering committee meeting that this would technically be a breaking change, since the attribute name is different. Could you revert this part of the change to keep the name `output_labeling_schema` here and in the text sections?